src/index.ts

```diff
@@ -160,5 +165,11 @@ require('dotenv').config()
     filterFile
   })
 
-  portman.run()
+  if (options.uploadOnly) {
+    // Upload only, skip all the rest
+    portman.uploadOnly()
```

Comment:
the _only_ thing I don't like is we skip the before method entirely, so we kill the console output, even though we still pass the config no?

Reply:
I commented out too much. the before() is back as part of the upload.

Reply:
hmm... in `uploadOnly` method? I don't see it?